core/src/main/scala/io/chrisdavenport/shellfish/SubProcess.scala

```diff
@@ -7,80 +28,113 @@ import fs2._
 import java.nio.file._
 import cats.effect.std.Supervisor
 
-
-/**
- * A Subprocess Approach To Running Shell Commands
- * Note: cp in this shell will do nothing, as the working directory is
- * determined by the Shell.
- **/
-trait SubProcess[F[_]]{
+/** A Subprocess Approach To Running Shell Commands Note: cp in this shell will
```

Comment:
nitpick: I personally prefer the previous comments style, We can keep it by adding `docstrings.style = Asterisk` to `.scalafmtAll`

Reply:
I also think that style is more curious to say at least! I can keep it on a quick commit